src/pages/feed.js

```diff
@@ -41,18 +44,9 @@ export default class Feed extends Component {
     await axios
       .post(process.env.REACT_APP_FEED, { counter: this.state.counter + 1 })
       .then((res) => {
+        const invalidIds = [1130, 1131, 1417, 1418, 1419, 641, 1547]
```

Comment:
This just cleanups up that stuff below. Also do we know why there's an invalid token_id? B/c we can cut this issue off at the source if so.

Reply:
These were IDs that were removed manually as people started to upload porn and stuff. were currently talking about how to block stuff...

Reply:
I've suggested to @crzypatchwork that we could manually add a tag to those "invalid ids" instead of polluting our source code with an array that will likely just keep growing. If we agree on a tag like `adult` or something like that, I can exlude the images coming in the feed that have that tag. and then even if someone randomly lands on that page I can hide that "adult" tag from the list of tags. so its not displayed on the OBJKT detail.

Reply:
But the tags that currently exist are onchain, right (in the ipfs md)? They can't be updated IMHO, means we need a list in any way to add the IDs to block... in the discord channel someone made a proposal, that would be better than hardcoding it...

Reply:
If you find it, do you mind sharing it here for visibility? I'm not sure if "we" (platform) can edit contracts, if we can't then we need a middle layer between the API request and the frontend response. and that should be server-side in my opinion.

Reply:
I'm also looking into separating the displays into a grid to show like 4 artworks on a row (2 on mobile, 3 on iPad). Is this something you guys are interested in @djangobits @andrevenancio ? Figure I should see if it's even desired before building it all :D

Reply:
Yeah I think that's one of the things we want to do. I have build a media query mixin that we should use, but its located at `src/styles/layout.scss`. and this is an example on how you should use it `src/components/media-types/video/index.module.scss` in there we define a mobile first layout then desktop (but we should also introduce the tablet). Ping me on discord if you need me to point you in the right direction.